cli/compose/convert/service_test.go

```diff
@@ -342,4 +342,38 @@ func TestConvertCredentialSpec(t *testing.T) {
 	})
 	assert.Error(t, err)
 	assert.Nil(t, swarmSpec)
+
+}
+
+func TestConvertUpdateConfigOrder(t *testing.T) {
+	// test default behavior
+	updateConfig := convertUpdateConfig(&composetypes.UpdateConfig{})
+	assert.Equal(t, "", updateConfig.Order)
+
+	// test start-first
+	updateConfig = convertUpdateConfig(&composetypes.UpdateConfig{
+		Order: "start-first",
+	})
+	assert.Equal(t, updateConfig.Order, "start-first")
+
+	// test stop-first
+	updateConfig = convertUpdateConfig(&composetypes.UpdateConfig{
+		Order: "stop-first",
+	})
+	assert.Equal(t, updateConfig.Order, "stop-first")
+}
+
+func TestConvertUpdateConfigParallelism(t *testing.T) {
+	parallel := uint64(4)
+
+	// test default behavior
+	updateConfig := convertUpdateConfig(&composetypes.UpdateConfig{})
+	assert.Equal(t, 1, updateConfig.Parallelism)
+
+	// Non default value
+	updateConfig = convertUpdateConfig(&composetypes.UpdateConfig{
+		Parallelism: &parallel,
+	})
+	assert.Equal(t, updateConfig.Parallelism, parallel)
```

Comment:
This needs to be `&parallel` (the test is failing), also the args are inverted, expected should be the second arg, and actual the third:

```go
assert.Equal(t, &parallel, updateConfig.Parallelism)
```